sass/components/_content_project.scss

```diff
@@ -0,0 +1,18 @@
+.ecl-description-list__definition {
```

Comment:
Please provide a comment header to explain what does this file contain in a sentence.
`/**`
` * @file`
` * This file is used to...`
 `*/`